Kajabi/sage-lib
# New Frame component (WIP)

## Description

Adds a new Frame component that provides Figma-like auto layout rules for creating ad hoc layouts and local components while using Sage components and tokens.

While the props and examples attempt to explain and give samples I believe that we can also add some demo videos and perhaps do a lunch and learn to raise awareness of this new tool.

## Screenshots

![Screen Shot 2022-04-28 at 9 37 28 AM](https://user-images.githubusercontent.com/17955295/165765083-ed957b6a-6080-48ae-b5be-39ea76916cf2.png)
![Screen Shot 2022-04-28 at 9 37 41 AM](https://user-images.githubusercontent.com/17955295/165765089-5f58fb2a-b3e3-443c-b7e9-ceda88b5ddf9.png)
![Screen Shot 2022-04-28 at 9 37 49 AM](https://user-images.githubusercontent.com/17955295/165765092-2a2b815e-cb42-4bf8-b533-d9c41073842b.png)
![Screen Shot 2022-04-28 at 9 37 57 AM](https://user-images.githubusercontent.com/17955295/165765095-823b2e8e-100d-49cf-98bd-7ed3b914f694.png)

![Screen Shot 2022-04-28 at 9 47 52 AM](https://user-images.githubusercontent.com/17955295/165767091-5b809d60-4f5b-48ce-bd72-4ac4912bb755.png)
![Screen Shot 2022-04-28 at 9 48 12 AM](https://user-images.githubusercontent.com/17955295/165767095-7211b57c-6754-4546-9ff2-95b7e8166b91.png)


## Testing in `sage-lib`

- See [Components > Frame](http://localhost:4000/pages/component/frame)
- See [feature in Storybook](http://localhost:4100/?path=/story/sage-frame--default)

## Testing in `kajabi-products`

1. (**N/A**) Added new SageFrame component.
